src/lib/sidebar/EntireScheme.svelte

```diff
@@ -132,53 +122,48 @@
   ).length;
 </script>
 
-<TextInput label="Scheme name" bind:value={$gjScheme.scheme_name} />
-
-<ErrorMessage {errorMessage} />
-
-<FileInput
-  label="Load from GeoJSON"
-  id="load-geojson"
-  disabled={$isAToolInUse}
-  {loadFile}
-/>
-
-<SecondaryButton on:click={exportToGeojson} disabled={$isAToolInUse}>
-  Export to GeoJSON
-</SecondaryButton>
-
-<div style="display: flex; justify-content: space-between">
-  <p>{$gjScheme.features.length} {schemaPluralNoun(schema)}</p>
-  <WarningButton
-    on:click={() => (displayClearAllConfirmation = true)}
-    disabled={$gjScheme.features.length == 0 || $isAToolInUse}
-  >
-    Clear all
-  </WarningButton>
-</div>
-<Modal
-  title="Would you like to clear your work?"
-  bind:open={displayClearAllConfirmation}
->
-  <p>This will delete all your drawn interventions.</p>
-  <div class="govuk-button-group">
-    <WarningButton on:click={clearAll}>Clear all work</WarningButton>
-    <SecondaryButton on:click={() => (displayClearAllConfirmation = false)}>
-      Cancel
+{#if $mode.mode == "list"}
+  <CollapsibleCard label={$gjScheme.scheme_name ?? "Untitled scheme"}>
+    <TextInput label="Scheme name" bind:value={$gjScheme.scheme_name} />
+
+    <ErrorMessage {errorMessage} />
+
+    <FileInput label="Load from GeoJSON" id="load-geojson" {loadFile} />
+
+    <SecondaryButton on:click={exportToGeojson}>
+      Export to GeoJSON
     </SecondaryButton>
-  </div>
-</Modal>
-{#if numErrors == 1}
-  <ErrorMessage errorMessage="There's a problem with one intervention below" />
-{:else if numErrors > 0}
-  <ErrorMessage
-    errorMessage="There's a problem with {numErrors} interventions below"
-  />
-{/if}
 
-{#if $isAToolInUse}
-  <p>
-    Finish drawing on the map and/or select "Edit attributes" to use these
-    options.
-  </p>
+    <div>
+      <WarningButton
+        on:click={() => (displayClearAllConfirmation = true)}
+        disabled={$gjScheme.features.length == 0}
+      >
+        Start new scheme
```

Comment:
Leaving as TODO, because I want to think through what the "scheme-wide" LCWIP fields will be and make the UI nicer. There's 3 buttons here now that are logically "New file", "Open file", and "Save file", but they don't really look that way now. If we move them all out, the left sidebar gets huge again and you can't see the list:
![Screenshot from 2023-10-21 13-59-38](https://github.com/acteng/atip/assets/1664407/e9e83e3e-7e6d-454c-b94a-cb77e97e816b)
and they're also not things people should really think about much. (Arguably that's true  for about / instructions / the title too)